playbook/app/pb_kits/playbook/pb_button/_button.jsx

```diff
@@ -22,7 +22,7 @@ type ButtonPropTypes = {
   loading?: boolean,
   newWindow?: boolean,
   onClick?: EventHandler,
-  size: 'large' | 'medium' | 'small',
+  size?: 'sm' | 'md' | 'lg',
```

Comment:
If this is being used somewhere in Nitro at the moment I would keep the existing props here until we migrated to the new prop names.

Reply:
I'm surprised that this prop was here but not on the Rails side though...

Reply:
This PR is a separate story and doesn't belong in here. It will be removed. Here's a PR that is adding a size prop to the button kit: https://github.com/powerhome/playbook/pull/1749

Reply:
Yeah, this is still a WIP and I'm going to need to revert those additions before attempting to merge this PR @haydenrou

Reply:
@haydenrou Fixed